tutorials/fundamentals/undefined-check.md

```diff
@@ -0,0 +1,29 @@
+In JavaScript, checking if a variable is undefined can be a bit tricky since
+a [null](/tutorials/fundamentals/null) variable can pass a check for undefined if not written properly.
+As a result, this allows for `undefined` values to slip through and vice versa.
+This is because `null` and `undefined` both mean nothing, so checking if they
+are equal `==` would return true. However, if you were to check if they were
+strictly equal `===`, it would return false as they are different ways of saying nothing.
+
+```javascript
+[require:Fundamentals good-undefined-check]
+```
+
+Objects are special because over the course of a program execution, properties
+can be assigned, removed, and added. As a result, they will not throw
+`ReferenceErrors` but instead return undefined.
+
+```javascript
+[require:Fundamentals undefined-objects]
+```
+
+You should not check if the variable is `undefined` but instead check if the `typeof` the variable is 'undefined'.
```

Comment:
I'd move this paragraph and example above the object one, because this one still has to do with variables. And with this one, I'd explain that `x === undefined` checks if `x` is a declared variable that is strictly equal to `undefined`. If you want to check whether `x` is strictly equal to `undefined` regardless of whether it has been declared or not, use `typeof x === 'undefined'`